src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -28,7 +28,7 @@
 long-summary: Certificates can be used as a secrets for provisioned virtual machines.
 examples:
   - name: Create a self-signed certificate with the default policy and add it to a virtual machine.
-    text: |
+    text: |-
```

Comment:
What is the `-` addition for?

Reply:
That is valid yaml syntax but I'll get rid of it